### PR TITLE
pkg/trace Enable stats endpoint

### PR DESCRIPTION
**Pull request by raphaelgavache**
Enable by default apm stats endpoint

### CODE REVIEW

File: pkg/trace/api/api_test.go [hunk 1]
```diff
@@ -615,46 +592,36 @@ func TestClientComputedStatsHeader(t *testing.T) {
...
+		case <-rcv.out:
+			wg.Wait()
+			return
+		case <-timeout:
+			t.Fatal("no output")
 		}
 	}
-
-	t.Run("on", run(true))
-	t.Run("off", run(false))
```
Comment (bmermet) [inline pkg/trace/api/api_test.go:657]: Why did you remove the test for when the clientComputedStats header is not set?

Comment (furmmon) [reply pkg/trace/api/api_test.go]: this is a mistake, changed the wrong test

**Review by bmermet — APPROVED** [review]

**Review by richardstartin — COMMENTED** [review]
Given that the payload accepted _isn't_ changing in a backward incompatible way, why are we changing the endpoint?
